src/_posts/2020-03-24-project-maturity-model.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+asset-type: post
+name: project-maturity-model
+title: Project Maturity Model
+date: 2020-02-24 10:00:00 +00:00
+author: Christopher Eyre
+image:
+    src: /assets/custom/img/blog/railway.png
+    attribution:
+        text: Public domain photo from Pixabay
+        href: https://pixabay.com/photos/rails-soft-gleise-railway-3309912/
+tags:
+    - project improvements
+abstract: A year of improvements to a project
+alias: [/2020/03/34/project-maturity-model]
+---
+
+## A Year in the life of a Project
+
+This article describes the evolution of a project over the course of a year. It has not been the full time focus of the team. 
+I want to describe the process in terms of the Capability Maturity Model. The CMM is intended for organisational effectiveness, but is easier to discuss in a smaller setting. 
+
+Here are the stages:
+
+1. Initial
+2. Managed
+3. Defined
+4. Quantitatively Managed
+5. Optimising
```

Comment:
I don't see an optimising section below

Reply:
There isn't one. I am not claiming that we had achieved perfection! In fact claiming level 4 is a bit of a stretch.

Reply:
Hahaha perhaps we should remove that? or maybe add a note that we didn't get there?